NineChronicles.Headless.Tests/GraphTypes/StandaloneQueryTest.cs

```diff
@@ -555,6 +556,81 @@ public async Task GetTx()
             Assert.Equal(transaction.Actions.First().PlainValue.Inspection, plainValue);
         }
 
+        [Fact]
+        public async Task Products()
+        {
+            var userPrivateKey = new PrivateKey();
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
+            StandaloneContextFx.BlockChain = service.Swarm.BlockChain;
+            var blockChain = service.BlockChain;
+
+            const string query = @"query {
+                products {
+                    sellerAgentAddress
+                    sellerAvatarAddress
+                    price
+                    itemUsable {
+                        itemId
+                        itemType
+                        itemSubType
+                    }
+                    costume {
+                        itemId
+                        itemType
+                        itemSubType
+                    }
+                }
+            }";
+            await blockChain.MineBlock(new Address());
+            var queryResult = await ExecuteQueryAsync(query);
+            var products = queryResult.Data.As<Dictionary<string, object>>()["products"].As<List<object>>();
+            Assert.True(products.Any());
+        }
+
+        [Theory]
+        [MemberData(nameof(ProductsMembers))]
+        public async Task ProductsWithArguments(int id, string itemSubType, int price, int expected)
+        {
+            var userPrivateKey = new PrivateKey();
+            var service = MakeMineChroniclesNodeService(userPrivateKey);
+            StandaloneContextFx.NineChroniclesNodeService = service;
```

Comment:
It seems useless to set `StandaloneContext.NineChroniclesNodeService ` in this scope.